src/UhdmAst.cpp

```diff
@@ -2726,10 +2726,11 @@ AstNode* visit_object(vpiHandle obj_h, UhdmShared& shared) {
                                    nullptr, nullptr);
         }
         visit_one_to_many({vpiArgument}, obj_h, shared, [&](AstNode* item) {
+            AstNode* argItem = new AstArg(new FileLine("Uhdm"), "", item);
```

Comment:
Please follow the convention required by Verilator for naming variables. That is use lowercase and `p` suffix if the variable is a pointer, so here it would be `argp` (instead of `argItem`)

(this convention has not been followed in this file from the very start and we will need to refactor it at some point, but let's try to stick to it at least with the new changes).

Reply:
Thanks. I didn't know about that convention. I fixed it.